test/parallel/test-net-connect-options-port.js

```diff
@@ -27,35 +27,37 @@ const net = require('net');
 
 // Test wrong type of ports
 {
-  function portTypeError(opt) {
-    const prefix = '"port" option should be a number or string: ';
-    const cleaned = opt.replace(/[\\^$.*+?()[\]{}|=!<>:-]/g, '\\$&');
-    return new RegExp(`^TypeError: ${prefix}${cleaned}$`);
+  function portTypeError() {
```

Comment:
@refack I have read the `CONTRIBUTING.md` but I am not sure which of the following should I use to make the requested changes:

1. amend the commit and push --force-with-lease
2. git commit --fixup
3. simply add a new commit